core/commands/storage.go

```diff
@@ -794,7 +796,13 @@ the shard and replies back to client for the next challenge step.`,
 		}
 		// build session
 		sm := storage.GlobalSession
-		ss := sm.GetOrDefault(ssID, n.Identity)
+		ss, err := sm.GetSession(n, storage.ShardsStorePrefix, ssID)
+		if err != nil {
+			log.Error(err)
```

Comment:
return err

Reply:
because the GetSession function return err when ssID not existed in the map, so we don't need to return err here.
but i'll add a TODO to refactor this.

Reply:
ok